<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogScanTaskInfo</class>
 <widget class="QDialog" name="DialogScanTaskInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑任务</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabTaskInfo">
      <attribute name="title">
       <string>任务信息列表</string>
      </attribute>
      <layout class="QHBoxLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>366</width>
            <height>370</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="taskInfoScrollLayout">
           <item row="0" column="0">
            <widget class="QGroupBox" name="taskInfo">
             <property name="title">
              <string>任务信息</string>
             </property>
             <layout class="QFormLayout" name="taskInfoLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="authorTitle">
                <property name="text">
                 <string>作者</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="author">
                <property name="text">
                 <string>author</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="createTimeTitle">
                <property name="text">
                 <string>创建时间</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="createTime">
                <property name="text">
                 <string>create</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="startTimeTitle">
                <property name="text">
                 <string>开始时间</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="startTime">
                <property name="text">
                 <string>start</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="endTimeTitle">
                <property name="text">
                 <string>结束时间</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="endTime">
                <property name="text">
                 <string>end</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="filePathTitle">
                <property name="text">
                 <string>文件路径</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLabel" name="filePath">
                <property name="text">
                 <string>filepath</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="taskOperatorTitle">
                <property name="text">
                 <string>操作员</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QLabel" name="taskOperator">
                <property name="text">
                 <string>operator</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0" colspan="2">
               <widget class="Line" name="line">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPlanInfo">
      <attribute name="title">
       <string>扫描计划</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,15,0" columnstretch="1,8">
       <item row="0" column="0">
        <widget class="QLabel" name="scanPlanName">
         <property name="text">
          <string>名称</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QVBoxLayout" name="scanPlan"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="name">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="exportPlan">
         <property name="text">
          <string>导出计划</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogScanTaskInfo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
